larray/tests/test_array.py

```diff
@@ -4134,6 +4134,16 @@ def test_dump():
                    ['a1', 1],
                    ['a2', 2]]
 
+    arr = ndtest((Axis('a0,a1'), Axis(2, 'b')))
+    res = arr.dump()
+    assert res == [['b', 0, 1],
```

Comment:
This is a misleading representation of the data (one would think that 'b' is the name of the column for 'a0' and 'a1'), so I don't really like this, but I am unsure we can do anything about it, except change our format (use two header lines instead of one), which is obviously out of scope here. **Maybe** that should come out as `\b` instead??? That would be ugly but non-misleading. Even **if** we go that route, if both axes are anonymous/empty, the `\` should of course not be shown.